### ISSUE TITLE
Explore: Loki data source does not show correctly logs volume for structured metadata label flitering

### ISSUE THREAD

**Issue by pyo-counting** [grafana/grafana#83204]
### What happened?

When i use logql with label filtering(`version` structured metatada), threre is no logs volumes for specific time range.
![1](https://github.com/grafana/grafana/assets/98924373/d1b4786e-f45b-4500-825a-a702a5e6f869)

And i zoomed time range to see that there is no logs really. But i can see that log and `version` structured metadata exists.
![306929427-9f9b4568-2c36-4034-b951-61b83c10a71c](https://github.com/grafana/grafana/assets/98924373/2b8754dd-8832-4069-bb5c-e356066e336a)
)

### What did you expect to happen?

show logs volume correctly in Grafana Explore tab

### Did this work before?

I've been testing grafana, loki for 3 days and it first happened today.
![3](https://github.com/grafana/grafana/assets/98924373/03240330-96fa-48b3-80c9-bbc0902c5111)

### How do we reproduce it?

1. Turn on structured metadata feature for Loki

### Is the bug inside a dashboard panel?

in Explore tab

### Environment (with versions)?

Grafana: v10.2.3 (1e84fede54)
OS: mac Monterey(12.7.1)
Browser: google chrome (122.0.6261.57 (arm64))

### Grafana platform?

Kubernetes

### Datasource(s)?

Loki 2.9.4

**Comment by pyo-counting** [comment]
@svennergr 
Thank you for your response. Unfortunately, we deleted the above loki instance and are running a new loki instance. And the above error is no longer occurring. If the above error occurs again, we will make sure to create an issue again with the query you requested.

**Comment by ivanahuckova** [comment]
I'll close this issue and feel free to reopen/create new one whenever issue reappears.

**Comment by LotisHunters** [comment]
I'm having the same issue.
The problem seems to be that Loki is no longer parsing out log level.
It seems to be pretty random. I restarted Loki on version 10, and it stopped getting that label. I upgraded to version 11, and they came back.
I restarted it a couple times and it was fine.
I restarted a few days later and the log levels disappeared again.

I can't make sense of how/why it randomly stops processing out the "level" label and I'm at a loss as to what to try.
![image](https://github.com/grafana/grafana/assets/47806082/71e30085-b99a-4529-8acc-e714fd2a57e3)


Here's an example from a few weeks ago (different application I know, but I can't get historical data from this one since I only store it in memory, and I just happened to have this screenshot)

![image](https://github.com/grafana/grafana/assets/47806082/e4a6be31-27fe-4d59-ad62-ce1234a9ad57)

That also reminds me that service_name is also no longer showing up as well.
